lib/net/http/request.rb

```diff
@@ -1,8 +1,26 @@
 # frozen_string_literal: false
-# HTTP request class.
-# This class wraps together the request header and the request path.
-# You cannot use this class directly. Instead, you should use one of its
-# subclasses: Net::HTTP::Get, Net::HTTP::Post, Net::HTTP::Head.
+
+# This class is the base class for \Net::HTTP request classes;
+# it wraps together the request path and the request headers.
+#
+# The class should not be used directly;
+# instead you should use its subclasses:
+#
+# - \Net::HTTP::Get
+# - \Net::HTTP::Head
+# - \Net::HTTP::Post
+# - \Net::HTTP::Delete
+# - \Net::HTTP::Options
+# - \Net::HTTP::Trace
+# - \Net::HTTP::Patch
+# - \Net::HTTP::Put
+# - \Net::HTTP::Copy
+# - \Net::HTTP::Lock
+# - \Net::HTTP::Mkcol
+# - \Net::HTTP::Move
+# - \Net::HTTP::Propfind
+# - \Net::HTTP::Proppatch
+# - \Net::HTTP::Unlock
```

Comment:
Should we remove the escaping so that it links to the subclasses?

Reply:
The subclasses' docs are empty(ish).